ocs_ci/utility/logging.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf8 -*-
+"""
+Module for logging related functions and classes.
+"""
+
+import logging
+import shutil
+
+from ocs_ci.framework import config
+from ocs_ci.ocs.constants import FILE_LOGGER, CONSOLE_LOGGER
+
+
+console_logger = logging.getLogger(CONSOLE_LOGGER)
+file_logger = logging.getLogger(FILE_LOGGER)
+
+
+class OCSLogFormatter(logging.Formatter):
+    def __init__(self):
+        fmt = (
+            "%(asctime)s - %(threadName)s - %(levelname)s -"
+            " %(name)s.%(funcName)s.%(lineno)d - %(message)s"
+        )
+        super(OCSLogFormatter, self).__init__(fmt)
+
+
+class CustomLoggerFilter(logging.Filter):
+    """
+    Logger filter which will filter out the specific logger messages we would
+    like to see in console output from pytest.
+    """
+
+    def filter(self, record):
+        """
+        Filter out only specific logs from tests folder, those which starts with
+        console.logger and ocs_ci.deployment.
+        """
+        log_all = config.REPORTING.get("log_all")
+        if log_all:
+            return FILE_LOGGER not in record.name
+        else:
+            return (
+                record.name.startswith("tests.")
+                or CONSOLE_LOGGER in record.name
+                or record.name.startswith("ocs_ci.deployment")
+            )
+
+
+def separator(symbol_, val=None):
+    """
+    Creates nice separator text which center the val surrounded by separator with width of terminal.
+
+    Args:
+        symbol_ (str): Symbol to use as separator, e.g. "-"
+        val (str): Value which should be in the center of the output text
+
+    Returns:
+        str: string like ------------- VAL ----------------
+
+    """
+    terminal_width = shutil.get_terminal_size(fallback=(120, 40))[0]
```

Comment:
IIUC in jenkins job it uses the fallback, right? I would propose to set it to lower number than 120 (what about 80?), because with the timestamp and other information in the beginning of the line it is quite long and on normal monitor is split to two lines, which doesn't look clear.
![obrazek](https://github.com/red-hat-storage/ocs-ci/assets/4759779/92ea50fa-0862-4d94-8af4-ad653f1f38bb)

Reply:
This is because the logger adding additional characters like:
`17:46:53 - MainThread - console.logger - INFO  -`
And then if we are adding more text in terminal size it's overflowing :(

I came up with solution that by default I am adding new line character before separated text.
So now it should add it to new line to match the size of terminal correctly on new line.